src/js/components/deployments/report.js

```diff
@@ -173,30 +177,10 @@ export class DeploymentReport extends React.Component {
       </Button>
     ];
 
-    var abort = (
-      <div className="float-right">
-        <Button
-          color="secondary"
-          onClick={() => self.setState({ abort: true })}
-          icon={<BlockIcon style={{ height: '18px', width: '18px', verticalAlign: 'middle' }} />}
-        >
-          Abort deployment
-        </Button>
-      </div>
-    );
-    if (self.state.abort) {
-      abort = <Confirm cancel={() => self.setState({ abort: false })} action={() => self._abortHandler()} type="abort" />;
-    }
-
-    var finished = '-';
-    if (deployment.finished) {
-      finished = <Time value={formatTime(deployment.finished)} format="YYYY-MM-DD HH:mm" />;
-    }
-
     return (
       <div>
-        <div className="report-container">
-          <div className="deploymentInfo" style={{ width: '240px', height: 'auto', margin: '30px 30px 30px 0', display: 'inline-block', verticalAlign: 'top' }}>
+        <div className={`report-container${past ? '' : ' report-progressing'}`}>
+          <div className="deploymentInfo" style={{ marginTop: 30 }}>
```

Comment:
Suggest removing marginTop – it seems to cause some misalignment

Reply:
ah I forgot this for the past deployments - this should only be applied for ongoing deployments, as it aligns with the abort button on the right... just waiting for my deployment to be picked up to verify the fix & then the PR is updated :)

Reply:
ah I was wrong, the marginTop was there for everything in the report - that should now align properly 👍